opentelemetry-otlp/examples/basic-otlp-http/Cargo.toml

```diff
@@ -15,6 +15,8 @@ bench = false
 [features]
 default = ["reqwest-blocking"]
 reqwest-blocking = ["opentelemetry-otlp/reqwest-blocking-client"]
+gzip = ["opentelemetry-otlp/gzip-http"]
```

Comment:
examples are currently kept basic.-showing absolute minimum required to work. I suggest to keep it that way in this PR. If we make a separate decision about improving these examples to show more scenarios, we can do it for this+ the gRPC examples too, in a separate PR.

Reply:
https://github.com/open-telemetry/opentelemetry-rust/blob/main/opentelemetry-otlp/src/lib.rs#L269-L271 The kitchensink would be a good place to showcase this.

Reply:
Done!